examples/src/pages/Migration.tsx

```diff
@@ -98,7 +99,7 @@ export function Migration() {
       (window as any).viewer = viewer;
 
       const controlsOptions: CameraControlsOptions = {
-        changeCameraTargetOnClick: true,
+        changeCameraTargetOnClick: false,
```

Comment:
If this is necessary and we cannot do this in a reasonably automatic way, it should be noted in the documentation that it's recommended to do this.